www/src/translation.ts

```diff
@@ -297,10 +301,12 @@ const dictionary: {[key: string]: DictionaryEntry} = {
     expedience: {
         en: "Expedience",
         ja: "匠の好機",
+        ko: "장인의 운",
```

Comment:
```suggestion
        ko: "장인의 운때",
```

https://guide.ff14.co.kr/job/Carpenter/21?type=L